service/resource/vaultcrt/delete.go

```diff
@@ -11,34 +11,12 @@ import (
 	"github.com/giantswarm/cert-operator/service/key"
 )
 
```

Comment:
* renamed `Process*State` -> `Apply*Change` and moved up
* renamed `Get*State` -> `new*Change` and moved down